src/ipyvizzustory/storylib/story.py

```diff
@@ -132,22 +132,43 @@ def add_step(self, step: Step) -> None:
 class StorySize:
     """A class for representing the size of a presentation story."""
 
-    def __init__(self, width: Optional[str] = None, height: Optional[str] = None):
+    def __init__(
+        self,
+        width: Optional[Union[int, str]] = None,
+        height: Optional[Union[int, str]] = None,
+        aspect_ratio: Optional[Union[float, str]] = None,
+    ):
         """
         StorySize constructor.
 
         Args:
             width: The width of a presentation story.
             height: The height of a presentation story.
+            aspect_ratio: The aspect-ratio of a presentation story.
+
+        Raises:
+            ValueError: If aspect_ratio and height are set together.
         """
+
+        width = str(width) + "px" if isinstance(width, int) else width
```

Comment:
check that `width` has 'px' suffix because otherwise `get_width_height_in_pixels` function will not work .

(it can be pass '111' string too as width/height, and the function will trim the last 2 character)

Reply:
fixed in [a8be36d](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/a8be36dd1a0fc35c92693e34228c27941cf07b26)